astropy/units/tests/test_equivalencies.py

```diff
@@ -520,11 +521,39 @@ def test_brightness_temperature():
     tb = 7.052590289134352 * u.K
     np.testing.assert_almost_equal(
         tb.value, (1 * u.Jy).to_value(
-            u.K, equivalencies=u.brightness_temperature(omega_B, nu)))
+            u.K, equivalencies=u.brightness_temperature(nu, beam_area=omega_B)))
     np.testing.assert_almost_equal(
         1.0, tb.to_value(
-            u.Jy, equivalencies=u.brightness_temperature(omega_B, nu)))
+            u.Jy, equivalencies=u.brightness_temperature(nu, beam_area=omega_B)))
 
+def test_beam():
+    omega_B = np.pi * (50 * u.arcsec) ** 2
+    new_beam = (5*u.beam).to(u.sr, u.equivalencies.beam_angular_area(omega_B))
+    np.testing.assert_almost_equal(omega_B.to(u.sr).value * 5, new_beam.value)
+    assert new_beam.unit.is_equivalent(u.sr)
```

Comment:
Just in case, also test the reverse...

Reply:
Also, I'd move this to after the other surface brightness tests.

Reply:
you mean `u.sr.is_equivalent(new_beam)`?

Reply:
I meant transform `new_beam` back to `u.beam` and check that you get back `5`.